portfolio/src/main/java/com/google/sps/data/Shows.java

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+/** An item on a todo list. */
+public final class Shows {
```

Comment:
Does a single instance of this class represent a single show or multiple shows?  If it is a single show, then typically you would see the class named "Show" instead of "Shows".  Having the javadoc a bit more descriptive would help the user understand this class.

Just by reading the class, it is not obvious to me what the fields represent.  One option is to add comments.  The other is to see if you can rename them.  So one example would be instead of calling the field "text", possibly call it "name" or "title".